.gitignore

```diff
@@ -39,3 +39,4 @@ compliance/repository/overlays
 /core/sail/elasticsearch-store/tempElasticsearchDownload/*
 tempElasticsearchDownload
 cache.db
+formatter-maven-cache.properties
```

Comment:
They for generated when I used maven with the formatter disabled. Since they didn’t go away with maven clean I decided to ignore them. So odd bug in the formatter I presume.